frontend/src/core/loaders/components/WaveLoader.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+import React from 'react';
+
+import './WaveLoader.css';
+
+export const WaveLoader = (): React.Node => (
+    <div id="project-load" className="overlay">
+        <div className="inner">
+            <div className="animation">
+                <div></div>
+                &nbsp;
+                <div></div>
+                &nbsp;
+                <div></div>
+                &nbsp;
+                <div></div>
+                &nbsp;
+                <div></div>
+            </div>
+            <div className="text">&quot;640K ought to be enough for anybody.&quot;</div>
```

Comment:
@mathjazz do you think we should make that string localizable? :thinking:

Reply:
I think it's not adding much value, so I'd just remove it for now.

Reply:
Actually, that's displayed before l10n strings are in, so there's no point localizing it.